draft-ietf-mmusic-sctp-sdp.xml

```diff
@@ -487,24 +515,28 @@
         prevent two separate SCTP associations from being established).
       </t>
       <t>
-        As both SCTP endpoints take the 'active' role, this specification does not define
-        usage of the SDP 'setup' attribute <xref target="RFC4145"/> for SCTP.
+        As both SCTP endpoints take the 'active' role, the SDP 'setup'
+        attribute <xref target="RFC4145"/> does not apply to SCTP association
+        establishment. However the 'setup' attribute does apply to establishment
+        of underlying DTLS associations and TCP connections.
       </t>
       <t>
         NOTE: The procedure above is different from TCP, where one endpoint takes the 'active'
         role, the other endpoint takes the 'passive' role, and only the 'active' endpoint initiates
         the TCP connection <xref target="RFC4145"/>.
       </t>
       <t>
-        NOTE: In case of SCTP-over-DTLS, when the SCTP association is established it is assumed
-        that any NAT traversal procedures for the underlying transport protocol (UDP or TCP) has
-        successfully been performed.
+        NOTE: When the SCTP association is established it is assumed that any NAT traversal
+        procedures for the underlying transport protocol (UDP or TCP) have successfully been
+        performed.
       </t>
       <t>
         Usage of the SDP 'connection' attribute <xref target="RFC4145"/> is not
```

Comment:
We use "define" also for other attributes, e.g., when talking about the 'setup' attribute in section 9.3.